src/include/migraphx/dynamic_loader.hpp

```diff
@@ -46,6 +47,7 @@ struct MIGRAPHX_EXPORT dynamic_loader
     static fs::path path(void* address);
 
     static optional<dynamic_loader> try_load(const fs::path& p);
+#endif
```

Comment:
Why is this disabled on windows?

Reply:
For some reason, on Windows, static analysis is showing the `catch` part of the function is unreachable code.  I made it available on Windows, too.